x-pack/plugins/enterprise_search/common/constants.ts

```diff
@@ -42,6 +42,22 @@ export const ENTERPRISE_SEARCH_CONTENT_PLUGIN = {
   SUPPORT_URL: 'https://discuss.elastic.co/c/enterprise-search/',
 };
 
+export const ESRE_PLUGIN = {
+  ID: 'esre',
```

Comment:
this should be a more unique id prefixed with `enterpriseSearch` since we need to reference outside of enterprise search plugin.

Reply:
Yeah also these ids are globally unique across Kibana 😬

Reply:
Renamed: https://github.com/elastic/kibana/pull/159589/commits/1bf101929fb22cc632601c16cabfb4e61952db33.